themes/bootstrap3/js/record.js

```diff
@@ -264,9 +264,128 @@ function applyRecordTabHash() {
   }
 }
 
+//VUFIND/EZBORROW INTEGRATION
+//WHEN THE LINK IS CLICKED -- CALL startPalciRequest
+//CHECKS TO SEE IF THE ITEM IS AVAIALABLE.
+$('#relaisRecordButton').click(startPalciRequest);
+
+Lightbox.addOpenAction(function() {
+  //REMOVE THIS?
+  //var serverResponse = event.srcElement.responseText;
+  //var icon = "spinning";
+  //alert(icon);
+  //if (serverResponse.indexOf(icon) !== -1) {
+  //  relaisRecordClickedFunction();
+  //}
+  //if ((serverResponse.indexOf(icon) !== -1) {
+ //  startPalciRequest();
+ // }
+});
+
+Lightbox.addCloseAction(function() {
```

Comment:
This is the deprecated logic of the previous iteration of the Lightbox. Now the Lightbox emits events. So you'll want to change this to
```JS
document.addEventListener("VuFind.lightbox.closing", function helpfulName() {
```
or maybe
```JS
$.on("VuFind.lightbox.closing", function helpfulName() {
```

Reply:
@crhallberg or @demiankatz should I use the exisiting ILLRequests/ILLRequestLink or should the EZ Borrow link be independent of the ILLRequest settings & code?

Reply:
@mis306lu, I would recommend implementing EZ Borrow independently of the existing ILL functionality; that is geared toward ILL functionality inside the ILS, and I could imagine a scenario where a library would want both -- a fast local option, and a slower regional one.

By the way, I've just invited you to join the developers group on this repository. If you accept, that should allow you to push changes directly to this branch if you wish to.